var/spack/repos/builtin/packages/log4cxx/package.py

```diff
@@ -14,14 +14,21 @@ class Log4cxx(CMakePackage):
 
     maintainers = ['nicmcd']
 
+    version('0.12.1', '807058734cebf7c92e99484313becbc6be63e48bd930889c733318bb78dfec498b66aaa12255ed30879abb005b32e2b965f9dd87ec50d0421ca067f3f66280d9')
     version('0.12.0', sha256='bd5b5009ca914c8fa7944b92ea6b4ca6fb7d146f65d526f21bf8b3c6a0520e44')
 
+    variant('std', default='17', description='C++ standard',
```

Comment:
This variant name should be `cxxstd` (as this is used across multiple packages)

Reply:
Thanks! Fixed